doc/scripts/make_images.py

```diff
@@ -12,6 +10,8 @@
 #
 
 def dataframe_line_plot(input_csv, output_svg):
+    import seaborn
```

Comment:
How does this fix the problem? It is still importing the problematic dependencies.

Instead, I thought that we had agreed to include the svg file in the documentation, and completely remove this code for generating the image.

Reply:
Yes, I left the function there just in case (it's not called). If you prefer, I can remove it entirely.

Reply:
Yes, let's remove it entirely.